prometheus-blackbox-exporter.tf

```diff
@@ -0,0 +1,111 @@
+locals {
+  prometheus-blackbox-exporter = merge(
+    local.helm_defaults,
+    {
+      name                   = "prometheus-blackbox-exporter"
+      namespace              = "monitoring"
+      chart                  = "prometheus-blackbox-exporter"
+      repository             = "https://prometheus-community.github.io/helm-charts"
+      create_ns              = false
+      enabled                = false
+      chart_version          = "4.10.3"
```

Comment:
Version `4.10.3` of this chart seems to be unavailable at this time (It looks like the release [failed in this CI run](https://github.com/prometheus-community/helm-charts/runs/1742259337) )

Version `4.10.2` is valid.

```console
$ helm search repo prometheus-community
...
prometheus-community/prometheus-blackbox-exporter 	4.10.2       	0.18.0     	Prometheus Blackbox Exporter                      
...
```